data/tld-!cn

```diff
@@ -468,6 +471,14 @@ teva # Teva Pharmaceutical Industries Limited
 toray # Toray Industries, Inc.
 weather # The Weather Channel LLC
 
+# Sex & Porn
+adult # Adult content
+gay # Homosexuality
```

Comment:
This one should be moved out too.

Reply:
Although I can move them to `Miscellaneous Domains`, the definition of sex here not only means pornography but also means sexuality.

Reply:
> Although I can move them to `Miscellaneous Domains`, the definition of sex here not only means pornography but also means sexuality.

I see. But I still get some strange felling about it.